include/api/CDataFrameTrainBoostedTreeClassifierRunner.h

```diff
@@ -76,9 +76,12 @@ class API_EXPORT CDataFrameTrainBoostedTreeClassifierRunner final
     inferenceModelDefinition(const TStrVec& fieldNames,
                              const TStrVecVec& categoryNames) const override;
 
-    //! \return A serialisable metadata of the trained regression model.
+    //! \return A serialisable metadata of the trained classification model.
     TOptionalInferenceModelMetadata inferenceModelMetadata() const override;
 
+    //! \return A serialisable data summarization for the trained classification model
+    TDataSummarizationUPtr dataSummarization(const core::CDataFrame& dataFrame) const override;
```

Comment:
I think this only needs to be declared and implemented on `CDataFrameTrainBoostedTreeRunner` no?

Reply:
You are right. Moved now.